index.html

```diff
@@ -4206,18 +4206,16 @@ <h4><a name="11cICP">11.3.3.6 <span class="chunk">cICP</span>
   <a href="#2-ITU-R-BT.709"><span class="NormRef">[ITU-R BT.709]</span></a>
   and <a href="#2-ITU-R-BT.2100"><span class="NormRef">[ITU-R BT.2100]</span></a></p>
 
-  <p>The cICP chunk contains three decimal values corresponding to
-  the colour primaries, transfer function and video signal range for the source imagery. </p>
+  <p>The cICP chunk contains four decimal values corresponding to
+  the colour primaries, transfer function, matrix coefficients and video signal range flag for the source imagery. </p>
 
-  <p>Note that Coding-Independent Code Points
-    <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a>
-    also define a fourth field, matrix coefficients,
-    which are used to describe
-    conversion of YCbCr or ICtCp image samples to an RGB color space.
-    However PNG image samples are explicitly RGB,
-    so the matrix coefficients are not defined in the cICP chunk.
-    Decoders should always assume a matrix coefficient value of 0
-    ("Identity") from the source.</p>
+  <p>Note that in Coding-Independent Code Points
+    <a href="#2-ITU-T-H.273"><span class="NormRef">[ITU-T H.273]</span></a>, the 
+    third video coding point defines matrix coefficients which are used to describe
+    conversion from other color representations to RGB. PNG image samples explicitly RGB,
```

Comment:
suggest adding (YCbCr, ICtCp) after "other color representations"

Reply:
Oh and in HTML  that should be

```html
YC<sub>b</sub>C<sub>r</sub>, IC<sub>T</sub>C<sub>P</sub>
```

Reply:
Fixed with: YC<sub>B</sub>C<sub>R/sub>, IC<sub>T</sub>C<sub>P</sub>.

Capitalized sub-script "B", and "R"